modules/setup/setup.js

```diff
@@ -1,83 +1,95 @@
 const Docker = require('dockerode');
 var docker = new Docker({ socketPath: '/var/run/docker.sock' });
 const child_process = require('child_process'),
-      IMAGE_TAG = 'tezsureinc/tezster:1.0.1',
+      IMAGE_TAG = 'tezsureinc/tezster:1.0.2',
       CONTAINER_NAME = 'tezster';
 
+const Logger = require('../logger');
 const { Helper } = require('../helper');
 
 class Setup {
 
     setup() {
-        console.log(Helper.outputInfo('We may need your password for write permission in config file....'));
+        Logger.verbose('Command : tezster setup');
+        Logger.info(Helper.outputInfo('We may need your password for write permission in config file....'));
 
         child_process.exec(`docker --version`, (error, _stdout, _stderr) => {
             if (_stdout.includes('Docker version')) {
                     this.checkPermission();
               } else {
-                    console.log(Helper.outputError('Docker not detected on the system please install docker....'));
+                    Logger.info(Helper.outputError('Docker not detected on the system please install docker....'));
               }
         });
     }
 
     startNodes() {
+        Logger.verbose('Command : tezster start-nodes');
         child_process.exec(`docker images ${IMAGE_TAG} --format '{{.Repository}}:{{.Tag}}:{{.Size}}'`,
         (error, _stdout, _stderr) => {
-            if (_stdout === `${IMAGE_TAG}:3GB\n`) {
+            if (_stdout === `${IMAGE_TAG}:2.96GB\n`) {
 
                 child_process.exec(`docker ps -a -q  --filter ancestor=${IMAGE_TAG} --format '{{.Image}}:{{.Names}}'`,
                 (error, _stdout, _stderr) => {
                     if (_stdout.includes(`${IMAGE_TAG}:${CONTAINER_NAME}\n`)) {
-                          console.log(Helper.outputInfo('Nodes are already running....'));
+                          Logger.info(Helper.outputInfo('Nodes are already running....'));
                     } else {
                           this.runContainer();
                     }
                 });
             } else {
-                  console.log(Helper.outputError(`No inbuilt nodes found on system. Run 'tezster setup' comamnd for build the nodes.`));
+                  Logger.info(Helper.outputError(`No inbuilt nodes found on system. Run 'tezster setup' comamnd for build the nodes.`));
```

Comment:
We had discussed to remove 'outpurError' functions and replace it with proper winston config.